processing/src/main/java/io/druid/segment/incremental/OnheapIncrementalIndex.java

```diff
@@ -39,14 +39,14 @@
 
 /**
  */
-public class OnheapIncrementalIndex extends IncrementalIndex<Aggregator>
+public class OnheapIncrementalIndex extends IncrementalIndex<Aggregator[]>
 {
-  private final ConcurrentHashMap<Integer, Aggregator[]> aggregators = new ConcurrentHashMap<>();
-  private final ConcurrentNavigableMap<TimeAndDims, Integer> facts = new ConcurrentSkipListMap<>();
-  private final AtomicInteger indexIncrement = new AtomicInteger(0);
-  protected final int maxRowCount;
+  private static final Logger log = new Logger(OnheapIncrementalIndex.class);
 
-  private String outOfRowsReason = null;
+  private final ConcurrentNavigableMap<TimeAndDims, Aggregator[]> facts = new ConcurrentSkipListMap<>();
+  private final int maxRowCount;
+
+  private transient String outOfRowsReason = null;
```

Comment:
I don't think "transient" is necessary, this object is not going to be serialized